packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -28,6 +27,8 @@ This rule also reports when an Array containing Promises is created and not prop
 - `Promise.any()`
 - `Promise.race()`
 
+Refer to [Handling Promises Rejections](#handling-promises-rejections) below for more information about this rule.
+
```

Comment:
I don't see any real value to including this. The page is tall; people will know that scrolling down will give them more information.

Reply:
I wouldn't make this assumption. I'm speaking for myself, but I often try to scan the page fastly and perform a search to get what I want. I don't read everything.

However, this is fine if I move the paragraph below the list, as suggested in the above.

Reply:
I moved the paragraph up and removed the phrase. Let me know.